ibis/backends/snowflake/tests/test_datatypes.py

```diff
@@ -0,0 +1,158 @@
+from __future__ import annotations
+
+import pytest
+from pytest import param
+
+import ibis
+import ibis.expr.datatypes as dt
+from ibis.backends.snowflake.datatypes import parse
+from ibis.backends.snowflake.tests.conftest import _get_url
+from ibis.util import gen_name
+
+dtypes = [
+    ("FIXED", dt.int64),
+    ("REAL", dt.float64),
+    ("TEXT", dt.string),
+    ("DATE", dt.date),
+    ("TIMESTAMP", dt.Timestamp(scale=9)),
+    ("VARIANT", dt.json),
+    ("TIMESTAMP_LTZ", dt.Timestamp(scale=9)),
+    ("TIMESTAMP_TZ", dt.Timestamp(timezone="UTC", scale=9)),
+    ("TIMESTAMP_NTZ", dt.Timestamp(scale=9)),
+    ("OBJECT", dt.Map(dt.string, dt.json)),
+    ("ARRAY", dt.Array(dt.json)),
+    ("BINARY", dt.binary),
+    ("TIME", dt.time),
+    ("BOOLEAN", dt.boolean),
+]
+
+
+@pytest.mark.parametrize(
+    ("snowflake_type", "ibis_type"),
+    [
+        param(snowflake_type, ibis_type, id=snowflake_type)
+        for snowflake_type, ibis_type in dtypes
+    ],
+)
```

Comment:
You could use `pytest.mark.parametrize(..., ids=callable)` to generate the labels, see https://docs.pytest.org/en/7.1.x/example/parametrize.html#different-options-for-test-ids